src/app/lib/settings.py

```diff
@@ -406,7 +406,7 @@ def load_settings() -> (
         worker: WorkerSettings = WorkerSettings()
 
     except ValidationError as e:
-        print("Could not load settings. %s", e)  # noqa: T201
+        print("Could not load settings.", e)  # noqa: T201
         raise e from e
```

Comment:
I think that you can also change `raise e from e` to `raise e`:

```python
>>> try:
...     raise ValueError(1)
... except ValueError as e:
...     raise TypeError(2)
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
TypeError: 2
>>> try:
...     raise ValueError(1)
... except ValueError as e:
...     raise TypeError(2) from e
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError: 1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
TypeError: 2
```

Reply:
@aorith do you want to make this change as well?

Reply:
Sure! A `raise from` doesn't make sense when re-raising the same exception!
I changed it to be only `raise` to avoid *TRY201*